### PR TITLE
Start Screen 

### PR DESCRIPTION
**Pull request by brandt19**
This update implements a start screen, start screen music and a button the user can press to bring them to the gameplay screen.

### CODE REVIEW

File: battleship/Form1.cs [hunk 1]
```diff
@@ -12,9 +12,26 @@ namespace battleship
...
+            musicPlayer.controls.play();
+        }
+
+        private void beginButton_Click(object sender, EventArgs e)
+        {
+            startScreen.Enabled = false;
+            startScreen.Visible = false;
+            beginButton.Enabled = false;
+            beginButton.Visible = false;
+            musicPlayer.controls.stop();
```
Comment (Ontrox) [inline battleship/Form1.cs:34]: Maybe we should play a game beginning fanfare or something here, but that is totally optional.

Comment (brandt19) [inline battleship/Form1.cs:34]: When the start screen pops up you mean?

Comment (Ontrox) [inline battleship/Form1.cs:34]: Just when the player clicks the begin button, to have an audio transition to the beginning of
 the game.

Comment (brandt19) [inline battleship/Form1.cs:34]: Yeah that would be easy to do. Right now it's just playing music during the
start screen and then it stops when the game begins. If you find an mp3
file you like I can put it in

**Pull request merged**